lib/embeds/widgets/default-embed-builder.dart

```diff
@@ -199,6 +200,13 @@ Widget defaultEmbedBuilder(
         readOnly: readOnly,
       );
 
+    case BlockEmbedM.formulaType:
+      final formula = node.value.data;
+      final style = node.style.attributes['style'];
+      print(node.style);
+      return Math.tex(formula, textStyle: Theme.of(context).textTheme.subtitle1);
```

Comment:
I guess we should not hardcode to one single style. At least the font size could be taken in account